app/templates/ui/app/create/create.html

```diff
@@ -62,9 +96,17 @@ <h2 class="col-sm-10">Create a Document</h2>
       </div>
     </div>
     <div class="form-group">
-      <label class="col-sm-2 control-label">Email</label>
+      <label class="col-sm-2 control-label required">Email</label>
       <div class="col-sm-10">
-        <input type="email" class="form-control" ng-model="ctrl.person.email" placeholder="">
+        <input type="email" class="form-control" ng-model="ctrl.person.email" placeholder="" name="email" required>
+      </div>
+    </div>
+    <div class="row form-error" ng-if="(createForm.$submitted || createForm.email.$touched) && createForm.email.$error">
+      <div class="col-md-10 col-md-offset-2 has-error has-feedback" >
+        <div ng-messages="createForm.email.$error">
+          <p class="alert alert-danger" ng-message="required">Email is required.</p>
```

Comment:
Could you drop required on email, and make eyeColor required instead? You can leave the email type check in place, but it should only fire if something was entered..